src/nbl/video/vulkan/profiles/NablaCore.json

```diff
@@ -1,2017 +1,2216 @@
-{
-    "$schema": "https://schema.khronos.org/vulkan/profiles-0.8.0-250.json#",
-    "capabilities": {
-        "baseline": {
-            "extensions": {
-                "VK_EXT_descriptor_indexing": 2,
-                "VK_EXT_external_memory_host": 1,
-                "VK_EXT_host_query_reset": 1,
-                "VK_EXT_image_robustness": 1,
-                "VK_EXT_inline_uniform_block": 1,
-                "VK_EXT_memory_budget": 1,
-                "VK_EXT_pipeline_creation_cache_control": 3,
-                "VK_EXT_private_data": 1,
-                "VK_EXT_robustness2": 1,
-                "VK_EXT_scalar_block_layout": 1,
-                "VK_EXT_separate_stencil_usage": 1,
-                "VK_EXT_shader_atomic_float": 1,
```

Comment:
need to diagnose why extensions went missing

Reply:
android